README.md

```diff
@@ -79,7 +79,8 @@ Invoke-Locksmith
 # Script Syntax
 .\Invoke-Locksmith.ps1
 ```
-Example Output for Mode 0: <https://github.com/TrimarcJake/Locksmith/blob/main/examples/Mode0.md>
+
```

Comment:
I don't like these blank lines between. Markdown formatting handles it fine. Can you remove?

Reply:
Yes